WordPress/src/main/res/values/dimens.xml

```diff
@@ -431,4 +431,7 @@
     <dimen name="web_preview_mode_popup_width">161sp</dimen> <!-- SP works better with popup list menu  -->
     <dimen name="web_preview_hint_label_corner_radius">6dp</dimen>
 
+    <!--settings username changer-->
+    <dimen name="settings_username_changer_confirm_dialog_adding">20dp</dimen>
```

Comment:
Typo: "adding" instead of "padding".

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10436/commits/01c8d819703b7a848b1c702ad46b98e59d03ce9e